datasets/ms_terms/README.md

```diff
@@ -0,0 +1,139 @@
+---
+YAML tags:
+annotations_creators:
+- expert-generated
+language_creators:
+- expert-generated
+languages: []
+licenses:
+- ms-pl
+multilinguality:
+- multilingual
+- translation
+size_categories:
+- 10K<n<100K
+source_datasets:
+- original
+task_categories: []
+task_ids: []
```

Comment:
what task is this dataset for ?

Reply:
It is for translation. It provides human translated terms for 100+ languages. There are around 100 files that can be downloaded. Each contains around 40K pairs between US English and the target language. One can also use the `term_id` to map between target languages. The downloadable files are all having `MicrosoftTermCollection.tbx` as its name. If a user only focuses on one language, it will be much simpler. Just download the .tbx file and then put it into a directory, and then load them. I selected translation task and expert curated in the tagging app, but the yaml file produced did not reflect it somehow.